docs/deployment/manage-projects.md

```diff
@@ -13,12 +13,12 @@ tags:
 View, sort, and tag your projects through the **Projects** page. Refer to this page to **manage** and **troubleshoot** thousands of repositories by identifying scan issues or scans with a high number of findings.
 
 :::note What is a project?
-A **project** is any repository that you have scanned with Semgrep AppSec Platform through CI or Semgrep Managed Scans. It includes that repository's findings and metadata.
+A **project** is a repository, or part of a repository, that you can scan through Semgrep AppSec Platform, either using CI or Semgrep Managed Scans. Its scans can be viewed on the **Project details** page, and its findings can be viewed on the individual Semgrep products' **Finding** pages.
 :::
 
 The **Projects** page features two components:
 
-- The **Scanning** page lists projects that are actively being scanned by Semgrep. These scans are either manually started by you, triggered by your CI/CD system, or automatically scanned by Semgrep Managed Scanning.
+The **Scanning** page lists all projects that have ever been provisioned or scanned by Semgrep, regardless of whether the project is actively being scanned. For active scans, they are either manually started by you, triggered by your CI/CD system, or automatically scanned by Semgrep Managed Scanning. This page also lists inactive projects *if* the repository hasn't been archived by the source code manager (SCM).
```

Comment:
The first sentence here makes sense, but I'm not sure what the rest is trying to say, other than making the archived distinction. I'd probably phrase that part differently to make it simpler: "If the project has been archived in the source code manager, it is listed under Not scanning". 
Though - @chmccreery At this point should it actually be "if the project's repository has been archived...", or how does that work?

Reply:
@armchairlinguist Sorry, I'm confused. Are you suggesting that this bullet point should be

> The **Scanning** page lists all projects that have ever been provisioned or scanned by Semgrep, regardless of whether the project is actively being scanned. If the project has been archived in the source code manager, it is listed under **Not scanning**".

?

Reply:
Basically, but with @chmccreery 's suggestion about project/repo too:

The **Scanning** page lists all projects that have been provisioned or scanned by Semgrep, regardless of whether the project is actively being scanned. If the project's repository has been archived in the source code manager, it is listed under **Not scanning**.

We could add something like "Managed scans do not run on archived repositories" if that helps clarify. It's also usually true of CI if the CI is integrated with the SCM, but otherwise I don't know that it's true, so probably better not to be specific there.